acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,645 @@
+    }
+    """
+    When method POST
+    Then status 201
+
+    # 2. Create package order-line for order
+    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+    * set poLine.id = poLineId2
+    * set poLine.purchaseOrderId = orderId2
+    * set poLine.isPackage = true
+    * set poLine.checkinItems = true
+    * set poLine.physical.createInventory = 'Instance, Holding, Item'
+    * set poLine.fundDistribution[0].fundId = fundId
+
+    Given path 'orders/order-lines'
+    And request poLine
+    When method POST
+    Then status 201
+
+    # 3. Create Title for this package order with acqUnit2, check that this POST Title operation is forbidden
+    * configure headers = headersUser
+    Given path 'orders/titles'
+    And request
+    """
+    {
+      id: "#(titleId1)",
+      title: "Sample Title",
+      poLineId: "#(poLineId2)",
+      "acqUnitIds": ['#(acqUnitId2)']
+    }
+    """
+    When method POST
+    Then status 403
+
+  Scenario: Repeat the same step with assigning acqUnit2 to user but make them inactive - POST Title operation should be also forbidden
+    # 'acqUnitId2' has already been created
+    # 1. Create acq unit membership
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/memberships'
+    And request
+    """
+      {
+        "id": '#(acqUnitMembershipId3)',
+        "userId": "00000000-1111-5555-9999-999999999992",
+        "acquisitionsUnitId": "#(acqUnitId2)"
+      }
+    """
+    When method POST
+    Then status 201
+
+    # 2. Make acqUnit2 inactive
+    Given path 'acquisitions-units/units', acqUnitId2
+    And request
+    """
+    {
+      "id": '#(acqUnitId2)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "isDeleted": true,
+      "name": "testAcqUnit2"
+    }
+    """
+    When method PUT
+    Then status 204
+
+    # 3. Verify POST Title to be forbidden
+    * configure headers = headersUser
+    Given path 'orders/titles'
+    And request
+    """
+    {
+      id: "#(titleId1)",
+      title: "Sample Title",
+      poLineId: "#(poLineId2)",
+      "acqUnitIds": ['#(acqUnitId2)']
+    }
+    """
+    When method POST
+    Then status 403
+
+  Scenario: Make acqUnit2 as active and repeat step again - operation should be allowed
+    # 1. Make acqUnit2 active
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/units', acqUnitId2
+    And request
+    """
+    {
+      "id": '#(acqUnitId2)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "isDeleted": false,
+      "name": "testAcqUnit2"
+    }
+    """
+    When method PUT
+    Then status 204
+
+    # 2. POST Title
+    * configure headers = headersUser
+    Given path 'orders/titles'
+    And request
+    """
+    {
+      id: "#(titleId1)",
+      title: "Sample Title",
+      poLineId: "#(poLineId2)"
```

Comment:
we should add here "acqUnitIds": ['#(acqUnitId2)']

Reply:
Thanks, added